applications/FluidDynamicsApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -180,6 +181,36 @@ void  AddCustomUtilitiesToPython(pybind11::module& m)
         .def("Execute", &AccelerationLimitationUtilities::Execute)
         ;
 
+    // Adding cfd utilities
+    m.def_submodule("CFDUtilities")
+        .def("CalculateNumberOfNeighbourConditions", &CFDUtilities::CalculateNumberOfNeighbourConditions)
+        .def("CalculateLinearLogarithmicWallFunctionBasedYPlusLimit", &CFDUtilities::CalculateLinearLogarithmicWallFunctionBasedYPlusLimit,
+            py::arg("von_karman") = 0.41,
```

Comment:
this is done this way to oblige using kwargs?

Reply:
I did this merely to avoid everytime passing the well known values for constants used in wall modelling. Does it has any effect on the performance?

Reply:
These python level calls are not used at the moment, I just exposed them to python in case someone wants to use them in their own scripts. If someone uses them, they need to call this method for every entity (node, element, condition). So I will remove them.